test-requirements.txt

```diff
@@ -7,3 +7,5 @@ vcrpy
 vcrpy-unittest
 unittest2 # For shopinvader test_controller, which inherits component
 odoo-test-helper
+# Restrict the version because of this PR on queue-job: https://github.com/OCA/queue/pull/403. That make test failed!
+odoo14-addon-queue-job<14.0.3.0.0
```

Comment:
I'm not sure is a good idea to fix it like this. 
No one will use this version in prod, we'll use the last one from qjob repo and if there's something broken w/ it we have to know it in advance. What's the problem w/ tests?

Reply:
Yes I don't know how to fix because it's a strange behavior (but that happens only if you have the last version of `queue_job`).
The issue is this: https://github.com/shopinvader/odoo-shopinvader/actions/runs/3051196438/jobs/4919136604#step:8:212

```
2022-09-14T08:20:31.8981553Z Traceback (most recent call last):
2022-09-14T08:20:31.8982112Z   File "/__w/odoo-shopinvader/odoo-shopinvader/shopinvader/tests/test_backend.py", line 145, in test_add_new_lang_rebind
2022-09-14T08:20:31.8982509Z     binded_variants_count * 2,
2022-09-14T08:20:31.8982776Z AssertionError: 74 != 146
```
If you don't use the last version of `queue_job`, you don't have the issue.

But I totally agree, the best should be to fix the source. And I don't have time right now to do it.

Reply:
ah but then we should have the fix here #1321 ;)
Can you cherry-pick @PierrickBrun 's commit here and rollback your change?

Reply:
Done :)
I didn't check in details #1321 when you ping me, so I didn't notice that was a fix related to the same issue.

Thank you for the review.
Hope this is the good one :muscle: